modules/prebidServerBidAdapter/index.js

```diff
@@ -232,12 +232,17 @@ function doClientSideSyncs(bidders) {
   });
 }
 
-function _getDigiTrustQueryParams() {
-  function getDigiTrustId() {
-    let digiTrustUser = window.DigiTrust && (config.getConfig('digiTrustId') || window.DigiTrust.getUser({member: 'T9QSFKPDN9'}));
+function _getDigiTrustQueryParams(bidRequest = {}) {
+  function getDigiTrustId(bidRequest) {
+    const bidRequestDigitrust = utils.deepAccess(bidRequest, 'bids.0.userId.digitrustid.data');
+    if (bidRequestDigitrust) {
+      return bidRequestDigitrust;
+    }
+
+    let digiTrustUser = (window.DigiTrust && (config.getConfig('digiTrustId') || window.DigiTrust.getUser({member: 'T9QSFKPDN9'})));
```

Comment:
Let's drop this from the PBS bid adapter -- shouldn't have been using our member number anyhow.

Reply:
I'll remove this from the PBS adapter.  Should this remain in the Rubicon adapter?

Reply:
Yes - stays in the RP adapter.

Reply:
Removed this in latest commit